src/applications/personalization/connected-accounts/components/ConnectedApp.jsx

```diff
@@ -52,14 +52,15 @@ class ConnectedApp extends React.Component {
                 Connected on {moment(created).format('MMMM D, YYYY h:mm A')}
               </th>
               <th className={`${cssPrefix}-row-details `}>
-                <a className={`${cssPrefix}-row-details-toggle`} href="#">
+                <button className={`${cssPrefix}-row-details-toggle`}
```

Comment:
Is the effort to refactor this table out into a list ( https://github.com/department-of-veterans-affairs/vets-contrib/issues/1438 ) on the near-term horizon? I think it will be a good idea to move the `onClick` handler to the button, but also appreciate the effort and time to do so. 

We have a link that's probably going to get removed wrapping the app name. I could see that coming back if the services had a user endpoint, or promo site, or something, so the right long-term approach is to wire event handlers to their keyboard focusable triggers.

I'll wait to hear what order things should be done. Otherwise I have no objections to this PR.

Reply:
ticket #1438 is coming up for us very soon, so it could go in there. but i think that i could also move it out there on this branch.

one question that's semi-related: do you have thoughts related to the other comment above? should the button look like a button or should it maintain the same visual presentation as before?

Reply:
@1Copenut let me know if that makes sense and if the new changes are good with you. if so, i'll merge the PR.

Reply:
@mjlumetta Cool, this looks good to me. I'm with Jeff on keeping the same visual treatment. We've done a bit of research, and as long as buttons announce and behave as buttons, and links as links, (not using a link with `role="button"` for instance) we're in good standing.